rkyv_test/Cargo.toml

```diff
@@ -1,19 +1,20 @@
 [package]
 name = "rkyv_test"
-version = "0.7.1"
-authors = ["David Koloski <djkoloski@gmail.com>"]
-edition = "2018"
 description = "Test suite for rkyv crates"
+version = "0.7.40"
+authors = ["David Koloski <djkoloski@gmail.com>"]
+edition = "2021"
 license = "MIT"
+documentation = "https://docs.rs/rkyv"
 repository = "https://github.com/rkyv/rkyv"
```

Comment:
Had to revert workspace definitions since `wasm-pack` is not able to parse workspace tables:
https://github.com/rustwasm/wasm-pack/blob/e3582b700217746ddb864909a20ef2f571a75326/src/manifest/mod.rs#L415-L423

Triggering these kinds of errors:
```
Error: failed to parse manifest: /home/runner/work/rkyv/rkyv/rkyv_test/Cargo.toml
Caused by: invalid type: map, expected a string for key `package.license`
```

Reply:
My bad, I should have reviewed the previous PR a little more carefully. Not your fault at all!